chiselFrontend/src/main/scala/chisel3/Module.scala

```diff
@@ -258,7 +258,7 @@ package experimental {
       *
       * @note Should not be called until circuit elaboration is complete
       */
-    @deprecated("toNamed API is deprecated -- use toTarget instead", "3.2")
+    //@deprecated(toNamed API is deprecated -- use toTarget instead", "3.3")
```

Comment:
Did you mean to delete the leading double quotes?

Reply:
Nope. Faulty regex. Thanks @jackkoenig.